### DISCUSSION TITLE
Why "supabase db reset --linked" sends local migrations to remote?

### DISCUSSION THREAD

**Discussion by tananga** [supabase/supabase#22604] [Questions]
I wonder why `supabase db reset --linked` commands automatically sends my local migrations into remote supabase instance? Remote have its own migrations which can be found in `supabase_migrations.schema_migrations`. I dont understand why `supabase db reset --linked` just dont reset remote database with remote migrations. I created a bug today related to this https://github.com/supabase/supabase/issues/22593. Maybe Im stupid or something but it just dont make any sense for me.

**Comment by pizofreude** [comment]
Have you tried to use the `supabase db reset` command without the `--linked` option?

**Reply by tananga** [reply]
Without --linked its gonna reset local db. I need to restart remote db without sending local migrations to remote.

**Reply by Goli4thus** [reply]
You can use `supabase migration list` to figure out the "first migration version".
Then use `supabase db reset --linked --version <first_migration_version>` to reset remote instance to only up to that version.

See `supabase db reset --help`.
